team/jensen-sun.yaml

```diff
@@ -0,0 +1,18 @@
+- title: Jensen (Ziheng) Sun
+  avatar: https://avatars.githubusercontent.com/u/17322987
+  role: Research Associate Professor
+  organizations:
+    - name: George Mason University
+  bio: I am a workflow guy and practitioner on using AI to address cool Earth scientific questions. 
```

Comment:
```suggestion
  bio: > 
    I am a workflow guy and practitioner on using AI to address cool Earth scientific questions. 
```

Reply:
https://stackoverflow.com/questions/3790454/how-do-i-break-a-string-in-yaml-over-multiple-lines/21699210#21699210